src/openssl/lib_crypto.cr

```diff
@@ -14,6 +14,10 @@
   end
 {% end %}
 
+# Check for brew's openssl libs on OS X
+{% if flag?(:darwin) %}
+  @[Link(ldflags: "-L/usr/local/opt/openssl/lib ")]
```

Comment:
Homebrew can be installed in many different locations, hardcoding a location inside crystal sourcecode isn't a valid fix.

Surely homebrew should be setting the linker and compiler env vars correctly to link to the correct place? `PATH` ,`LIBRARY_PATH`, `CPATH` etc?